docs/community-operators-troubleshooting.md

```diff
@@ -0,0 +1,219 @@
+The summary of failed static tests will be posted to the Pull Request as a comment with detailed reasons. Fix all of 
+failed static tests and retrigger the hosted pipeline. To learn more about static tests, please follow this [documentation](https://redhat-openshift-ecosystem.github.io/operator-pipelines/users/static_checks/).
+
+## <a id="static-tests-results"></a>static-tests-results
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="merge-registry-credentials"></a>merge-registry-credentials
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="digest-pinning"></a>digest-pinning
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="verify-pinned-digest"></a>verify-pinned-digest
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="dockerfile-creation"></a>dockerfile-creation
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="build-bundle"></a>build-bundle
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="make-bundle-repo-public"></a>make-bundle-repo-public
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="get-supported-versions"></a>get-supported-versions
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="add-bundle-to-index"></a>add-bundle-to-index
+Failures at this step are uncommon and if they do occur they are often transient.  
+
+Please click the `Close pull request` button in GitHub then click the `Reopen pull request` button.
+Closing and re-opening your Pull request will restart the Pipeline. If your PR fails at this step twice in a row please contact Red Hat Support
+
+## <a id="make-index-repo-public"></a>make-index-repo-public
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="get-ci-results-attempt"></a>get-ci-results-attempt
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="preflight-trigger"></a>preflight-trigger
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+> Note: There is a known issue if you Operator only supports OpenShift 4.7 or below. In this case we recommend using the CI Pipeline.
+
+## <a id="evaluate-preflight-result"></a>evaluate-preflight-result
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
```

Comment:
This is not true. In case a preflight test fails it is because the operator didn't pass the tests. We should tell a user where to find preflight logs and resources.